src/ansiblelint/yaml_utils.py

```diff
@@ -225,6 +233,211 @@ def _nested_items_path(
             )
 
 
+def get_path_to_play(
+    lintable: Lintable,
+    line_number: int,  # 1-based
+    ruamel_data: Union[CommentedMap, CommentedSeq],
+) -> List[Union[str, int]]:
+    """Get the path to the play in the given file at the given line number."""
+    if lintable.kind != "playbook" or not isinstance(ruamel_data, CommentedSeq):
+        return []
+    # TODO: raise if line_number < 1 or just return [] ?
```

Comment:
This is one open question I have. If for some reason, someone passes a `line_number < 1`, then they're probably passing in an index, not a number which is wrong.

Should I raise an error? Or just `return []` because anything no play or task can be at lines less than 1?

Reply:
I just made it raise ValueError. If that ends up being annoying, we can change it in a follow-up.